plugins/module_utils/_ADObject.psm1

```diff
@@ -1016,6 +1026,13 @@ Function Invoke-AnsibleADObject {
         }
     )
 
+    if ($ModuleNoun -eq "ADGroup") {
```

Comment:
This should not be part of the shared module util function but an option inside the group module itself. Unfortunately it probably means that the logic for the `members` option needs to split out of the util.

Reply:
Can you explain this in more detail? I don't really understand what to do here.